### PR TITLE
Update dependencies

### LINKED ISSUE #77: Update dependencies
Please update this to only use hapi dependencies from the new `@hapi/` scoped modules. This will help me keep it used in vision tests. Let me know if you need help.

### PR DESCRIPTION
**Pull request by jedireza**
Specifically for `@hapi` scoped modules. Resolves #77.

### CODE REVIEW

**Comment by jedireza** [comment]
@hueniverse want to give this a quick review?

File: examples/layout/server.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+
 const Hapi = require('hapi');
```
Comment (hueniverse) [inline examples/layout/server.js]: Don't forget these...

File: examples/remount/server.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+
 const Path = require('path');
 const Hapi = require('hapi');
```
Comment (hueniverse) [inline examples/remount/server.js]: And these.

File: examples/simple/server.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+
 const Hapi = require('hapi');
```
Comment (hueniverse) [inline examples/simple/server.js]: Etc.

**Comment by jedireza** [comment]
Thank you.

Published as `10.0.1`.

**Pull request merged**